views/pages/index.ejs

```diff
@@ -92,7 +92,7 @@
           <div class="col-lg-5 col-md-12 col-sm-12">
             <div class="contents text-center">
               <h1 class="wow fadeInDown" data-wow-duration="1000ms" data-wow-delay="0.3s">Bakhu Holdings</h1>
-              <p class="lead  wow fadeIn" data-wow-duration="1000ms" data-wow-delay="400ms">Lorem ipsum dolor sit amet, consectetur adipiscing elit. </p>
+              <!-- <p class="lead  wow fadeIn" data-wow-duration="1000ms" data-wow-delay="400ms">Lorem ipsum dolor sit amet, consectetur adipiscing elit. </p> -->
```

Comment:
git commits will keep history. No need to leave comments in. This can be removed.